### ISSUE TITLE
Cannot use this app on non-Google Android phones because of Google services

### ISSUE THREAD

**Issue by dexter-2** [joachimneumann/bisqremote_Android#12]
Due to the reliance on Google services, this app cannot be used on non-Google Android devices.

I procured the SDK for my Android LineageOS phone (Aurora Store, F-Droid) but upon open the Pair button is greyed out, and a few seconds later the following popup message appears:

>Downloading notification token...
>Are you connected to the internet?

Note: I did not press the Pair button at any point.

My device is connected to the internet, and I can press the Learn More button to prove this.

It's worth noting that this issue would not be that important except for the presumption that Bisq has a relatively significant number of users who own non-Google Android phones due to its nature as an anonymous and decentralised method of exchanging Bitcoin and other cryptos.

**Comment by kaschmirrr** [comment]
Yes, exactly the same problem here. Running GrapheneOS on Google Pixel 4 (no Google services). I have seen many Bitcoiners who use GrapheneOS (and other privacy focused degoogled Android phones), so it would be nice if this could be solved.

**Comment by devinbileck** [comment]
Link to further discussion: https://github.com/bisq-network/bisq/issues/2441

**Comment by devinbileck** [comment]
Threema just implemented push notifications on Android without FCM.
Reference:
- https://threema.ch/en/blog/posts/threema-push
- https://github.com/threema-ch/threema-android/commit/3f2dc16873c775f60acf2a8580a0d249a4d857f8

**Comment by geekley** [comment]
https://f-droid.org/2022/12/18/unifiedpush.html

**Comment by heronimoo** [comment]
I agree that a non-Google notification alternative would be a substantial improvement to this app. Using Unified Push like @geekley suggested also appears to me like the most promising solution. [ntfy](https://github.com/binwiederhier/ntfy) for instance could be, from what I understand, configured as a drop-in replacement for Google's Firebase Cloud Messaging (the standard Google notification system). For lazy users, notifications would be routed via ntfy's servers - and more advanced users could self-host ntfy to be completely independent.
